exampleCourse/questions/demo/autograder/c/salesTax/info.json

```
@@ -11,7 +11,7 @@
     "externalGradingOptions": {
         "enabled": true,
         "image": "prairielearn/grader-c",
-        "timeout": 1000,
+        "timeout": 100,
```

This is a pretty quick code, 100 seconds is overkill.

```suggestion
        "timeout": 10,
```